Dockerfile

```diff
@@ -14,6 +14,10 @@ COPY todo_app todo_app
 
 ENV FLASK_APP=todo_app/app
 
+ENV FLASK_ENV=development
+
+ENV SECRET_KEY=secret-key
```

Comment:
Setting a default is a little odd for an argument that is meant to be secret, but could make sense (worth considering the trade-off between convenience for dev/testing, vs the risk that we accidentally rely on the default in production leaving us less secure!)